_drafts/oryx/packages.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Oryx packages
+description: Use Oryx packages from npm to ensure you can easily upgrade to newer versions.
+last_updated: Apr 19, 2023
+template: concept-topic-template
+---
+
+While the Oryx code base is [available on Github](https://github.com/spryker/oryx/), the code is published and distributed as so-called packages on npm. [npmjs.com](https://www.npmjs.com/) is a widely used registry that is used to packages. Package managers (e.g. npm, yarn, deno, bun, etc) can be used to install the dependencies in a project. The dependencies are typically configured in the [package.json](https://docs.npmjs.com/cli/v9/configuring-npm/package-json) file of your application.
```

Comment:
Why are the packages `so-called`?

Reply:
we can leave that out. I'm often in conversations with folks who are not so into the node module ecosystem, and need some intro to "packages".